src/python/analyzer_executor/src/run.py

```diff
@@ -23,28 +25,31 @@ async def _main() -> None:
     - add the shortcut-to-DEAD_LETTER_QUEUE_URL case
     - pull the manual eventing out of `handle_events` and into an EventEmitter (maybe?)
     """
-    queue_url = os.environ["SOURCE_QUEUE_URL"]
-    retriever = EventRetriever(queue_url=queue_url)
-    sqs_client = SQSClientFactory(boto3).from_env()
-    analyzer_executor = AnalyzerExecutor.from_env()
-
-    for sqs_message in retriever.retrieve():
-        # We'd feed this coroutine into the timeout manager.
-        message_handle_coroutine = analyzer_executor.handle_events(sqs_message.body, {})
-        # While we're waiting for that future to complete, keep telling SQS
-        # "hey, we're working on it" so it doesn't become visible on the
-        # input queue again.
-        timeout_manager = SqsTimeoutManager(
-            sqs_client=sqs_client,
-            queue_url=queue_url,
-            receipt_handle=sqs_message.receipt_handle,
-            message_id=sqs_message.message_id,
-            coroutine=message_handle_coroutine,
-        )
-        try:
-            await timeout_manager.keep_alive()
-        except SqsTimeoutManagerException:
-            LOGGER.error("SQS Timeout Manager exception", exc_info=True)
+    with TRACER.start_as_current_span("_main"):
```

Comment:
Might be better to use `_main.__name__` instead of `"_main"` for this span name, just to guard against refactoring-induced errors in the future.